### ISSUE TITLE
Problem with sample data?

### ISSUE THREAD

**Issue by steveharman** [DavidBelicza/magento2-server-configuration#7]
HI,

Getting this on install....

Thanks,

Steve

```INSTALLATION: MAGENTO 2 SAMPLE DATA DEMO STORE
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - The requested package magento/module-grouped-product-sample-data could not be found in any version, there may be a typo in the package name.
  Problem 2
    - The requested package magento/module-sales-sample-data could not be found in any version, there may be a typo in the package name.
  Problem 3
    - The requested package magento/module-cms-sample-data could not be found in any version, there may be a typo in the package name.
  Problem 4
    - The requested package magento/module-customer-sample-data could not be found in any version, there may be a typo in the package name.
  Problem 5
    - The requested package magento/module-swatches-sample-data could not be found in any version, there may be a typo in the package name.
  Problem 6
    - The requested package magento/module-review-sample-data could not be found in any version, there may be a typo in the package name.
  Problem 7
    - The requested package magento/sample-data-media could not be found in any version, there may be a typo in the package name.
  Problem 8
    - The requested package magento/module-offline-shipping-sample-data could not be found in any version, there may be a typo in the package name.
  Problem 9
    - The requested package magento/module-bundle-sample-data could not be found in any version, there may be a typo in the package name.
  Problem 10
    - The requested package magento/module-theme-sample-data could not be found in any version, there may be a typo in the package name.
  Problem 11
    - The requested package magento/module-tax-sample-data could not be found in any version, there may be a typo in the package name.
  Problem 12
    - The requested package magento/module-downloadable-sample-data could not be found in any version, there may be a typo in the package name.
  Problem 13
    - The requested package magento/module-sales-rule-sample-data could not be found in any version, there may be a typo in the package name.
  Problem 14
    - The requested package magento/module-catalog-sample-data could not be found in any version, there may be a typo in the package name.
  Problem 15
    - The requested package magento/module-configurable-sample-data could not be found in any version, there may be a typo in the package name.
  Problem 16
    - The requested package magento/module-product-links-sample-data could not be found in any version, there may be a typo in the package name.
  Problem 17
    - The requested package magento/module-widget-sample-data could not be found in any version, there may be a typo in the package name.
  Problem 18
    - The requested package magento/module-msrp-sample-data could not be found in any version, there may be a typo in the package name.
  Problem 19
    - The requested package magento/module-wishlist-sample-data could not be found in any version, there may be a typo in the package name.
  Problem 20
    - The requested package magento/module-catalog-rule-sample-data could not be found in any version, there may be a typo in the package name.

Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see <https://getcomposer.org/doc/04-schema.md#minimum-stability> for more details.

Read <https://getcomposer.org/doc/articles/troubleshooting.md> for further common problems.

Installation failed, reverting ./composer.json to its original content.```

**Comment by DavidBelicza** [comment]
Hello,

Thanks for the report.

I guess the user/key auth is not active in your Magento Connect account. This is why it can not see the Magento's private composer packages.

David

**Comment by MrDaar** [comment]
Yes, and not really related to this script. Shall we close this ticket?

**Comment by steveharman** [comment]
I confirmed my Magento credentials worked fine (public & private keys for accessing repo.magento.com).  Anything else I need to verify?

Thanks

**Comment by DavidBelicza** [comment]
It can not be reproduced. Maybe this is fixed issue in the new version #22
